pfunk/__main__.py

```diff
@@ -9,12 +9,16 @@
 from __future__ import absolute_import, division
 from __future__ import print_function, unicode_literals
 
+from itertools import product
+
 import argparse
 import fnmatch
 import os
 import subprocess
 import sys
 
+import multiprocessing as mp
```

Comment:
can go in the list above (alphabetically)

Reply:
Also in general disapprove of aliases: unless they definitely improve readability, they're just one more thing to remember when reading the code

Reply:
Fixed.